### DISCUSSION TITLE
How to load environment variable?

### DISCUSSION THREAD

**Discussion by himself65** [wakujs/waku#287] [Q&A] [answered]
I'm adding server actions with database connection

**Comment by dai-shi** [comment]
I think @sandren did some tricks in `packages/website`. We don't consider it a final solution.

**Reply by himself65** [reply]
https://github.com/dai-shi/waku/blob/54867a213ba9db93ba31d43b52d5255f4626e619/packages/website/vite.config.ts#L4

**Comment by himself65** [comment, marked as answer]
https://github.com/dai-shi/waku/pull/321
